### ISSUE TITLE
[Feature] show up or down color of candle bar based on previous close and current close

### ISSUE THREAD

**Issue by manchingchan1992** [klinecharts/KLineChart#495]
### Feature Description

Can add a new candle type to show up or down color of candle bar based on previous close and current close.  Currently up or down color depends on current close > current open or current close < current open

### To Do

CandleBarView add logic to check up or down color based on previous close >current close  or previous close < current close.

**Comment by dgandram** [comment]
Definitely, this is a must. I'm just beginning to use klinecharts so I don't know how to do it but I will try to implement it and if I'm able to, I will do a PR.
